packages/features/eventtypes/components/AddMembersWithSwitch.tsx

```diff
@@ -14,18 +15,21 @@ interface IUserToValue {
   id: number | null;
   name: string | null;
   username: string | null;
-  avatar: string;
+  avatar: string | null;
   email: string;
+  accepted?: boolean;
 }
 
 export const mapUserToValue = (
-  { id, name, username, avatar, email }: IUserToValue,
+  { id, name, username, avatar, email, accepted }: IUserToValue,
   pendingString: string
-) => ({
+): TeamMember => ({
   value: `${id || ""}`,
-  label: `${name || email || ""}${!username ? ` (${pendingString})` : ""}`,
-  avatar,
+  // Showing Pending badge if user has not accepted
+  label: `${name || email || ""}${!username && accepted !== false ? ` (${pendingString})` : ""}`,
```

Comment:
Not sure where this was being used

Reply:
I think changing the common UI component is not a good idea.
we can instead pass the component prop to the <Select/>
take a look how option is implementd

https://github.com/SomayChauhan/cal.com/blob/f3cb6d28a91244c1416b56667517aa6d4e012b8d/apps/web/components/eventtype/EventAvailabilityTab.tsx#L31